packages/amplify-migration-tests/src/__tests__/migration_tests/api.connection.migration.test.ts

```diff
@@ -0,0 +1,85 @@
+import { initJSProjectWithProfile, deleteProject, amplifyPush, amplifyPushUpdate } from '../../../../amplify-e2e-tests/src/init';
+import { addApiWithSchema, updateApiSchema } from '../../../../amplify-e2e-tests/src/categories/api';
+import { createNewProjectDir, deleteProjectDir } from '../../utils';
```

Comment:
Move this to core and use it in both e2e and migration test. Lets not have multiple copies of the same functionality